### PR TITLE
document tourniquet models

### PR DESCRIPTION
**Pull request by hdefazio**
- Documentation
- Renamed current tourniquet object detector to match documentation
- Added new tourniquet models too anisble

### CODE REVIEW

File: ansible/roles/provision-files/vars/main.yml [hunk 1]
```diff
@@ -56,15 +56,28 @@ girder_file_downloads:
...
       sha512: e3d44527b572cb95f1869ec33214cbaf9c721aae18f14bc9db73ea8ed5c9c9d6c52975ebc0884acd3402e8ec66fcfd6168ff5744c5a9e31069049d23a97c5fce
-      dest: "{{ stage_dirs.model_files }}/recipe_m2_apply_tourniquet_v0.052.pkl"
+      dest: "{{ stage_dirs.model_files }}/recipe_m2_apply_tourniquet_v0.052.pkl" 
+    # Tourniquet object detection
+    - file_id: 645a94db58fefcae81a9f73a
+      sha512: 2f07f3ec2759c729d70d89cd31266e000226208fbdcf25e50018cbb6c85e5c5a51378b73049334d5472f8dfe000040dbc740e65e093c6cd114d83e96bfcdab6d
+      dest: "{{ stage_dirs.model_files }}/tourniquet_stage1.pth"
+    - file_id: 645a946058fefcae81a9f737
+      sha512: 7382a33fe69bdb97ae218a30a5797404ef6573637a6bf28e04041f7c87b056bbb1d3e2dfb6ea8ceb869a6d73aff9e53e0d11a43e62e7c132964394bd0e033ff8
+      dest: "{{ stage_dirs.model_files }}/tourniquet_stage2_interstitial.pth"
```
Comment (Purg) [inline ansible/roles/provision-files/vars/main.yml:79]: are these replacing the manually handed out `m2_with_lab_cleaned_fixed_data_with_*.pth` weights files?

File: docs/algorithms/training.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            'M2-29','M2-41', 'M2-1' 'M2-127'   'M2-23', 'M2-17' 'M2-139', 'M2-55', 'M2-69' 'M2-68', 'M2-13',
+            'M2-133', 'M2-121', 'M2-14', 'M2-50', 'M2-44', 'M2-43', 'M2-35', 'M2-75', 'M2-37']
+    - BBN's M2 Lab Data (HoloLens videos):  ['tq_12', 'tq_8', 'tq_21', 'tq_23', 'tq_3', 'tq_20', 'tq_19',
+            'tq_14', 'tq_13', 'tq_10', 'tq_11', 'tq_5']
+
+
+Validation data:
+    - BBN's M2_Tourniquet v0.52 videos: ['M2-72', 'M2-58', 'M2-134', 'M2-105', 'M2-33', 'M2-9', 'M2-42',
+            'M2-30', 'M2-16', 'M2-128']
+    - BBN's M2 Lab Data (HoloLens videos): ['tq_18', 'tq_22']
```
Comment (Purg) [inline docs/algorithms/training.rst:29]: I don't see the value of this info being separated out over here as opposed to just being under the relevant models described in the `docs/algorithms/model_versions.rst` file.

**Review by Purg — COMMENTED** [review]
This seems to be targetting ansible provisioning of models we are using for the BBN demo, so I would like to target the BBN release branch with this, and reflect this on their machine. Please target the `release-bbn-integration` branch and rebase on that branch as well.

File: ansible/roles/provision-files/vars/main.yml (hunk 1 above)
Comment (hdefazio) [reply ansible/roles/provision-files/vars/main.yml:79]: yes - I figured since it is an officially tested model I could clean up the name now

**Review by Purg — APPROVED** [review]

**Pull request merged**